packages/cursorless-org-docs/src/docs/contributing/CONTRIBUTING.md

```diff
@@ -139,6 +139,19 @@ To uninstall the local build and revert to production cursorless, run the follow
 pnpm -F cursorless-vscode uninstall-local
 ```
 
+## Regular manual maintenance tasks
+
+### Updating the cheatsheet
+
+We need to keep the default cheatsheet up-to-date for use with local development and for https://www.cursorless.org/cheatsheet.
+
+1. Switch to vanilla knausj + cursorless_talon
+2. "debug stench"
+3. "cursorless update cheatsheet"
```

Comment:
I don't have this option

Reply:
https://github.com/cursorless-dev/cursorless/blob/e5e9929c05e2607cc7bbddf1d22013b5aa89c772/cursorless-talon-dev/src/cursorless_dev.talon#L32-L33

Reply:
Ah I thought it was a launch option. Got it.